core/kernel/impl/class.ApiModelOO.php

```diff
@@ -133,58 +120,44 @@ public function getResourceDescriptionXML($uriResource)
+                    $root->setAttribute("xmlns:$namespaceId", $namespace);
                 }
 
-                $resourceValue = false;
-                foreach ($allNs as $namespaceUri) {
-                    if (
-                        preg_match('/^' . preg_quote($namespaceUri, '/') . '/', $object) ||
-                        preg_match("/^http:\/\/(.*)#[a-zA-Z1-9]*/", $object)
-                    ) {
-                        $resourceValue = true;
-                        break;
+                $namespaceId = $namespaces[$namespace];
+                $nodeName    = "$namespaceId:$property";
+
+                try {
+                    $node = $dom->createElement($nodeName);
+                    if (!empty($lang)) {
+                        $node->setAttribute('xml:lang', $lang);
                     }
-                }
-                if (!is_null($nodeName)) {
-                    try {
-                        $node = $dom->createElement($nodeName);
-                        if (!empty($lang)) {
-                            $node->setAttribute('xml:lang', $lang);
-                        }
-
-                        if ($resourceValue) {
-                                $node->setAttribute('rdf:resource', $object);
-                        } else {
-                            if (!empty($object) && !is_null($object)) {
-
-                                /**
-                                 * Replace the CDATA section inside XML fields by a replacement tag:
-                                 * <![CDATA[ ]]> to <CDATA></CDATA>
-                                 * @todo check if this behavior is the right
-                                 */
-                                $object = str_replace(['<![CDATA[', ']]>'], ['<CDATA>', '</CDATA>'], $object);
-
-                                $node->appendChild($dom->createCDATASection($object));
-                            }
-                        }
-                        $description->appendChild($node);
-                    } catch (DOMException $de) {
-                        //print $de;
+
+                    if (preg_match("/^http:\/\/(.*)#[a-zA-Z1-9]*/", $object)) {
+                        $node->setAttribute('rdf:resource', $object);
+                    } elseif (!empty($object) && !is_null($object)) {
+
+                        /**
+                         * Replace the CDATA section inside XML fields by a replacement tag:
+                         * <![CDATA[ ]]> to <CDATA></CDATA>
+                         * @todo check if this behavior is the right
+                         */
+                        $object = str_replace(['<![CDATA[', ']]>'], ['<CDATA>', '</CDATA>'], $object);
+
+                        $node->appendChild($dom->createCDATASection($object));
                     }
+                    $description->appendChild($node);
+                } catch (DOMException $de) {
+                    //print $de;
```

Comment:
I know that it was like this before, but could you please at least log the error?